### ISSUE TITLE
Changes in structure between releases and no doc!

### ISSUE THREAD

**Issue by loicgasser** [mtgjson/mtgjson#226]
Hi,
I am using MTG json (for which I donated money) to help me build my website and I noticed that you introduced **A LOT** of changes to cards structure. It's a bit annoying when you have automated updates like me and that you realize that something as important as `cmc` has been renamed to `convertedManaCost` without any further notice in the releases.

I was wondering if you could just add the breaking changes in the release notes for those like me that use it.
Just a few lines would be great! Keep up the good work!

**Comment by ZeldaZach** [comment]
Hey @loicgasser Thanks for the donation to the project!

With MTGJSONv4, we made several breaking changes to the project compared to the v3 release (Which is why we incremented the major number). 

If you look in the 4.0 release notes (found at https://mtgjson.com/changelog) you'll see a full list of all the changes we've made. Included in this list is the following:

> Changed
> `cmc` is now `convertedManaCost`

We never had a `castingCost` field, so I assume that's referring to `manaCost`? 

Anyway, if you come across any other issues, do reach out!
